api/src/main/java/com/codeforcommunity/dto/school/School.java

```diff
@@ -0,0 +1,50 @@
+package com.codeforcommunity.dto.school;
+
+import com.codeforcommunity.enums.Country;
+
+public class School {
+
+  private Long id;
```

Comment:
I still think we should switch over to UUID: 
1. If we have to backfill data into a new model, that new model can't use an autoinc ID, as it must accept the existing schools. A UUID can be generated independent of one specific DB table, while generating an autoinc ID requires the context of our DB system. 
2. Autoinc IDs give more properties than the ID is meant to - essentially, it gives away things like chronological order of creation, when in reality it should be purely a unique ID. We're storing created at anyways. 

Again, a lot of this is because half of my coop is nuking autoinc IDs and replacing them with UUIDs... 

This doesn't specifically apply to schools, and I won't require any action. 

https://medium.com/@Mareks_082/auto-increment-keys-vs-uuid-a74d81f7476a

Reply:
(1) I'm still not sure why we would need to worry about IDs when backfilling data? If we just work with `INSERT` statements and leave ID generation to the database, what's the issue with using autoinc IDs? (Why do we have to worry about pre-defining IDs outside of the context of our database?)
(2) Yes auto incrementing IDs give away things like ordering of creation and table size--but I don't suspect that would be any issue with our proj--we're not doing anything where we would have to worry about such a "leak" of information for our `schools`.

That being said, I think we should revisit this discussion before backfilling data for the MVP. At this point, it would require a migration to change either way, so there's not a big advantage to doing that now rather than in another week or two.